scripts/idl/tests/outputs/cluster_struct_attribute/cpp/BridgeClustersImpl.h

```diff
@@ -0,0 +1,148 @@
+#include <app-common/zap-generated/cluster-id.h>
+#include <app-common/zap-generated/attribute-id.h>
+
+#include <new>
+
+namespace clusters {
+
+struct DemoClusterCluster : public CommonCluster
```

Comment:
I think we should split the cluster struct from clusterinfo and attrinfo.

I believe that code would be easier to maintain if the generated code looks somewhat similar to what one would manually generate (minus constants or some formatting).